docs/style-guides/js.md

```diff
@@ -0,0 +1,252 @@
+# origin-web-console JS style guide
+
+A reasonable approach to assisting in style consistency in `openshift-web-console`
+inspired by guides such as the style guide published by [airbnb](https://github.com/airbnb/javascript).
+
+## ESLint
+
+Run `grunt eslint` to check code for errors and style discrepancies.
+
+## Examples
+
+
+### Variables
+
+Group your variable names but use individual `var` statements:
```

Comment:
I suggest we go with @jwforres suggestion. Unless anyone disagrees, I'd add not to declare vars inside an `if` block since they're function scoped anyway.